activity/manager.go

```diff
@@ -67,7 +67,10 @@ func New(idleTimeout, stopRetryPeriod time.Duration) (Manager, error) {
 
 	workspaceName, isFound := os.LookupEnv("CHE_WORKSPACE_NAME")
 	if !isFound {
-		return nil, errors.New("CHE_WORKSPACE_NAME env must be set for activity manager works correctly")
+		workspaceName, isFound = os.LookupEnv("DEVWORKSPACE_NAME")
+		if !isFound {
+			return nil, errors.New("CHE_WORKSPACE_NAME env or DEVWORKSPACE_NAME env must be set for activity manager works correctly")
```

Comment:
I know it's me who suggested to who in the following way, but after thinking a bit more, I think it makes sense to go in the following way https://github.com/eclipse-che/che-machine-exec/pull/142/commits/1d8368735633b1dacc1b6a944ce7409e80fd117c since activityTracker won't work in che mode.

After I pushed changes, I realized that it won't work for WTO. So, reverting.